bdd/spec/011_array_spec.sh

```diff
@@ -401,9 +401,9 @@ true"
 
           print('reduce as filter and concat test')
           // TODO: Lots of improvements needed for closures passed directly to opcodes. This one-liner is ridiculous
-          test.reduce(fn (acc: string, i: int): string = (acc == '' && i % 2 == 1) ? i.toString() : (i % 2 == 1 ? (acc + ', ' + i.toString()) : acc), '').print()
```

Comment:
This change is a bit horrifying to me, because I don't understand how it compiled correctly before looking at the operator precedence table.

But that also makes me wonder if the precedence levels have been set correctly. The previous version looks better than this one.